pkg/k8sutil/pod.go

```diff
@@ -0,0 +1,24 @@
+package k8sutil
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// PodUsesPVC returs true if provided pod has provided pvc among its volumes.
+func PodUsesPVC(pod corev1.Pod, pvc corev1.PersistentVolumeClaim) bool {
```

Comment:
Nit: funcs that return a bool usually has as prefix has or is. wdyt about rename this one?

Reply:
No problem, would you mind suggesting a name for it ?

Reply:
By following the convention any fun that returns a bool usually is named with HasSomething or IsSomething 
So, I'd recommend here HasPVC wdyt? Would that make sense?

Reply:
Yeah, I understand. I think solely `HasPVC` lacks a little bit of context like "what" has a PVC. Maybe `PodHasPVC` ? But then why not `Uses` instead of `Has` (it makes more sense to me).

Reply:
function renamed to `HasPVC`.